### PR TITLE
Add a third-party scheduler, selectable from one of the kraken-services folders

### PR DESCRIPTION
**Pull request by maratoid**

### CODE REVIEW

**Comment by maratoid** [comment]
This will most likely not pass PR checks without https://github.com/Samsung-AG/kraken-services/pull/76 being merged first

**Comment by spiffxp** [comment]
retest this please

File: ansible/roles/kubernetes/defaults/main.yaml [hunk 1]
```diff
@@ -7,8 +7,9 @@ hyperkube_deployment_mode: binary
 hyperkube_image: gcr.io/google_containers/hyperkube:v1.1.1
 kraken_services_branch: master
 # TODO: can we avoid the skydns hardcode and just specify it here?
-kraken_services_dirs: "heapster kube-ui prometheus"
+kraken_services_dirs: "skydns cluster-monitoring kube-ui prometheus"
```
Comment (spiffxp) [inline ansible/roles/kubernetes/defaults/main.yaml]: this satisfies the TODO, let's remove it

File: ansible/roles/kubernetes/templates/kraken-delete-services.sh.ansible [hunk 2]
```diff
@@ -1,7 +1,12 @@
 #! /usr/bin/bash
+#! /usr/bin/bash
```
Comment (spiffxp) [inline ansible/roles/kubernetes/templates/kraken-delete-services.sh.ansible]: dupe

**Pull request merged**